templates/docs/rhpam78-authoring-ha.adoc

```diff
@@ -84,6 +84,8 @@ https://docs.okd.io/latest/architecture/core_concepts/templates.html#parameters[
 |`BUSINESS_CENTRAL_CPU_LIMIT` | -- | Business Central Container CPU limit. | 2000m | True
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE server Container memory limit. | 1Gi | True
 |`KIE_SERVER_CPU_LIMIT` | -- | KIE server Container CPU limit. | 1000m | True
+|`BUSINESS_CENTRAL_CONTAINER_REPLICAS` | -- | Business Central Container Replicas, defines how many Business Central containers will be started. | 2 | True
```

Comment:
just a minor comment, for future changes on templates.
there's no need to update the adocs, it is auto generated for every new release :)